docs/website/data/integrations.yaml

```diff
@@ -293,8 +293,10 @@ integrations:
       - kong
     code:
       - https://github.com/TravelNest/kong-authorization-opa
+      - https://github.com/open-policy-agent/contrib/tree/master/kong_api_authz
     inventors:
       - travelnest
+      - wada-ama
```

Comment:
I believe we'll need a `wada-ama` entry under `organizations` further down in the file

Reply:
Ok, entry added!